specification/2.0/Specification.adoc

```diff
@@ -1407,32 +1696,26 @@ Samplers are stored in the `samplers` array of the asset. Each sampler specifies
 }
 ----
 
-[NOTE]
-.Default Filtering Implementation Note
-====
-When filtering options are defined, runtime must use them. Otherwise, it is free to adapt filtering to performance or quality goals.
-====
 
-[NOTE]
-.Mipmapping Implementation Note
-====
-When a sampler's minification filter (`minFilter`) uses mipmapping (`NEAREST_MIPMAP_NEAREST`, `NEAREST_MIPMAP_LINEAR`, `LINEAR_MIPMAP_NEAREST`, or `LINEAR_MIPMAP_LINEAR`), any texture referencing the sampler needs to have mipmaps, e.g., by calling GL's `generateMipmap()` function.
-====
+==== Non-power-of-two Textures
+
+Client implementations **SHOULD** resize non power-of-two textures when running on certain platforms that have limited support for such texture dimensions.
 
 [NOTE]
-.Non-Power-Of-Two Texture Implementation Note
+.Implementation Note
 ====
-glTF does not guarantee that a texture's dimensions are a power-of-two.  At runtime, if a texture's width or height is not a power-of-two, the texture needs to be resized so its dimensions are powers-of-two if the `sampler` the texture references
+Specifically if the `sampler` the texture references
```

Comment:
This is a bit context-free. It could repeat the point: 

> Specifically, client implementations **SHOULD** resize non-poper-of-two textures (so that their horizontal and vertical sizes are powers of two) if the `sampler` the texture references ...